BraveRewardsUI/Ads/AdContentButton.swift

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import BraveRewards
+import pop
+
+/// The main ads view. Mimics a system notification in that it shows an icon, "app name" (always will be "Brave Rewards"), title and body.
+class AdContentButton: UIControl {
+  let titleLabel = UILabel().then {
+    $0.textColor = .black
+    $0.font = .systemFont(ofSize: 15.0, weight: .semibold)
+    $0.numberOfLines = 2
+  }
+  let bodyLabel = UILabel().then {
+    $0.textColor = UIColor(white: 0.0, alpha: 0.5)
+    $0.font = .systemFont(ofSize: 15.0)
+    $0.numberOfLines = 3
```

Comment:
No hard limit, just wanted to keep it from growing too tall. iOS system notifications I believe is a 2 line limit, we decided on 3 since our format's a bit smaller